services/src/main/java/org/apache/druid/cli/CliMiddleManager.java

```diff
@@ -195,6 +200,18 @@ private void configureIntermediaryData(Binder binder)
             biddy.addBinding("deepstore").to(DeepStorageIntermediaryDataManager.class).in(LazySingleton.class);
           }
 
+          @Provides
+          @LazySingleton
+          @Named(ServiceStatusMonitor.HEARTBEAT_TAGS_BINDING)
+          public Supplier<Map<String, Object>> heartbeatDimensions(WorkerConfig workerConfig, WorkerTaskManager workerTaskManager)
+          {
+            return () -> ImmutableMap.of(
+                "workerVersion", workerConfig.getVersion(),
```

Comment:
Should we try to make the dimensions as few as possible? Means to have a general dimension name so it can be reused in other services, e.g. `workerVersion` -> `version`, `workerCategory` -> `category`

Reply:
In this instance I think more specific dimension names are helpful as it clears up some confusion. eg. `version` can be misinterpreted - Should it be the version of Druid that it is running or version of the worker.

If we find a dimension that should be reported on multiple services, this would be good to consider

Reply:
Thanks to this comment, I did look for standard dimension names that are used by other metrics and used static references to those dimensions. LMK if this looks better